mslice/plotting/plot_window/cut_plot.py

```diff
@@ -100,16 +100,14 @@ def update_legend(self, line_data=None):
     def change_axis_scale(self, xy_config):
         current_axis = self._canvas.figure.gca()
         if xy_config['x_log']:
-            xdata = [ll.get_xdata() for ll in current_axis.get_lines()]
-            xmin = get_min(xdata, absolute_minimum=0.)
+            xmin = xy_config['x_range'][0]
```

Comment:
`symlog` should be able to handle a negative scale <= 0 so what I think we need to do here is set the min/max scale to the true values specified by the user in `xy_config` but then use `get_min(xdata, absolute_minimum=0.)` to calculate the number that goes into `linthreshx` and not use `xmin` there.

This should gives us a log scale that is linear below the minimum data values.